src/main/java/dk/sdu/mmmi/modulemon/HeadlessBattleView/HeadlessBattleView.java

```diff
@@ -62,19 +68,39 @@ public class HeadlessBattleView implements IGameViewService {
     private int[] battleAmounts = {1, 10, 100, 250, 500, 750, 1000};
     private int battleAmountIndex = 1;
 
+    private CSVWriter csvWriter;
+
     @Override
     public void init(IGameViewManager gameViewManager) {
         selectSound = AssetLoader.getInstance().getSoundAsset("/sounds/select.ogg", this.getClass());
         chooseSound = AssetLoader.getInstance().getSoundAsset("/sounds/choose.ogg", this.getClass());
         yaySound = AssetLoader.getInstance().getSoundAsset("/sounds/YAY.ogg", this.getClass());
         battleWaitMusic = AssetLoader.getInstance().getMusicAsset("/music/headless.ogg", this.getClass());
         battleWaitMusic.setLooping(true);
-        battleWaitMusic.setVolume(getSoundVolume());
+        battleWaitMusic.setVolume(getMusicVolume());
         menuMusic = AssetLoader.getInstance().getMusicAsset("/music/headlessMenu.ogg", this.getClass());
         menuMusic.setLooping(true);
-        menuMusic.setVolume(getSoundVolume());
+        menuMusic.setVolume(getMusicVolume());
         menuMusic.play();
 
+        csvWriter = new CSVWriter();
+        csvWriter.setWriteExcelHeader(true); // Makes it so the file is easy to open in Excel.
+
+        var coloumnTitles = new ArrayList<String>() {{
+            add("Turn");
```

Comment:
Should be battle instead.
```suggestion
            add("Battle");
```